build.sbt

```diff
@@ -697,6 +720,76 @@ lazy val contribs = (project in file("contribs"))
     Compile / compile := ((Compile / compile) dependsOn createTargetClassesDir).value
   ).configureUnidoc()
 
+
+val unityCatalogVersion = "0.3.0"
+val sparkUnityCatalogJacksonVersion = "2.15.4" // We are using Spark 4.0's Jackson version 2.15.x, to override Unity Catalog 0.3.0's version 2.18.x
```

Comment:
Unrelated:  I think we may need to relocate the `Jackson` in the oss-unitycatalog spark connector, since it's such a general client-side jar, which will run under different runtime environments. And for those common jars,  it can be easily conflicted with the jars introduced from other projects. 

I had an issue for oss-unitycatalog before: https://github.com/unitycatalog/unitycatalog/issues/1141